Epoch 1/30


59/59 [==============================] - ETA: 0s - loss: 0.6507 - mean_absolute_error: 0.0845
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
59/59 [==============================] - 6s 76ms/step - loss: 0.6507 - mean_absolute_error: 0.0845 - val_loss: 0.6340 - val_mean_absolute_error: 0.0390
Epoch 2/30

59/59 [==============================] - ETA: 0s - loss: 0.6264 - mean_absolute_error: 0.0369
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
59/59 [==============================] - 8s 137ms/step - loss: 0.6264 - mean_absolute_error: 0.0369 - val_loss: 0.6326 - val_mean_absolute_error: 0.0333
Epoch 3/30

59/59 [==============================] - ETA: 0s - loss: 0.6253 - mean_absolute_error: 0.0331
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
59/59 [==============================] - 4s 72ms/step - loss: 0.6253 - mean_absolute_error: 0.0331 - val_loss: 0.6319 - val_mean_absolute_error: 0.0315
Epoch 4/30

59/59 [==============================] - ETA: 0s - loss: 0.6248 - mean_absolute_error: 0.0320
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 70ms/step - loss: 0.6248 - mean_absolute_error: 0.0320 - val_loss: 0.6314 - val_mean_absolute_error: 0.0298
Epoch 5/30
33/59 [===============>..............] - ETA: 1s - loss: 0.6261 - mean_absolute_error: 0.0317
59/59 [==============================] - 3s 46ms/step - loss: 0.6250 - mean_absolute_error: 0.0335 - val_loss: 0.6323 - val_mean_absolute_error: 0.0348
Epoch 6/30

59/59 [==============================] - ETA: 0s - loss: 0.6245 - mean_absolute_error: 0.0316
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 72ms/step - loss: 0.6245 - mean_absolute_error: 0.0316 - val_loss: 0.6308 - val_mean_absolute_error: 0.0295
Epoch 7/30
11/59 [====>.........................] - ETA: 2s - loss: 0.6232 - mean_absolute_error: 0.0298

59/59 [==============================] - ETA: 0s - loss: 0.6238 - mean_absolute_error: 0.0290
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 71ms/step - loss: 0.6238 - mean_absolute_error: 0.0290 - val_loss: 0.6306 - val_mean_absolute_error: 0.0279
Epoch 8/30
 7/59 [==>...........................] - ETA: 2s - loss: 0.6181 - mean_absolute_error: 0.0291

53/59 [=========================>....] - ETA: 0s - loss: 0.6233 - mean_absolute_error: 0.0284
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 70ms/step - loss: 0.6236 - mean_absolute_error: 0.0284 - val_loss: 0.6302 - val_mean_absolute_error: 0.0258
Epoch 9/30
 5/59 [=>............................] - ETA: 2s - loss: 0.6232 - mean_absolute_error: 0.0330


58/59 [============================>.] - ETA: 0s - loss: 0.6246 - mean_absolute_error: 0.0326
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
59/59 [==============================] - 4s 69ms/step - loss: 0.6244 - mean_absolute_error: 0.0326 - val_loss: 0.6302 - val_mean_absolute_error: 0.0269
Epoch 10/30
59/59 [==============================] - 3s 47ms/step - loss: 0.6236 - mean_absolute_error: 0.0289 - val_loss: 0.6318 - val_mean_absolute_error: 0.0354
Epoch 11/30

59/59 [==============================] - ETA: 0s - loss: 0.6235 - mean_absolute_error: 0.0287
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 70ms/step - loss: 0.6235 - mean_absolute_error: 0.0287 - val_loss: 0.6299 - val_mean_absolute_error: 0.0259
Epoch 12/30
31/59 [==============>...............] - ETA: 1s - loss: 0.6228 - mean_absolute_error: 0.0278

59/59 [==============================] - ETA: 0s - loss: 0.6232 - mean_absolute_error: 0.0275
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 70ms/step - loss: 0.6232 - mean_absolute_error: 0.0275 - val_loss: 0.6299 - val_mean_absolute_error: 0.0256
Epoch 13/30
27/59 [============>.................] - ETA: 1s - loss: 0.6233 - mean_absolute_error: 0.0284
59/59 [==============================] - 3s 48ms/step - loss: 0.6232 - mean_absolute_error: 0.0276 - val_loss: 0.6303 - val_mean_absolute_error: 0.0279
Epoch 14/30

59/59 [==============================] - 3s 46ms/step - loss: 0.6234 - mean_absolute_error: 0.0289 - val_loss: 0.6301 - val_mean_absolute_error: 0.0269
Epoch 15/30
59/59 [==============================] - 3s 45ms/step - loss: 0.6230 - mean_absolute_error: 0.0263 - val_loss: 0.6320 - val_mean_absolute_error: 0.0354
Epoch 16/30

59/59 [==============================] - ETA: 0s - loss: 0.6239 - mean_absolute_error: 0.0304
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 69ms/step - loss: 0.6239 - mean_absolute_error: 0.0304 - val_loss: 0.6297 - val_mean_absolute_error: 0.0242
Epoch 17/30
19/59 [========>.....................] - ETA: 1s - loss: 0.6272 - mean_absolute_error: 0.0265
59/59 [==============================] - 3s 48ms/step - loss: 0.6228 - mean_absolute_error: 0.0257 - val_loss: 0.6298 - val_mean_absolute_error: 0.0262
Epoch 18/30


59/59 [==============================] - ETA: 0s - loss: 0.6228 - mean_absolute_error: 0.0257
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
59/59 [==============================] - 4s 70ms/step - loss: 0.6228 - mean_absolute_error: 0.0257 - val_loss: 0.6297 - val_mean_absolute_error: 0.0251
Epoch 19/30
59/59 [==============================] - ETA: 0s - loss: 0.6226 - mean_absolute_error: 0.0250
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
59/59 [==============================] - 4s 70ms/step - loss: 0.6226 - mean_absolute_error: 0.0250 - val_loss: 0.6294 - val_mean_absolute_error: 0.0233
Epoch 20/30
59/59 [==============================] - 3s 47ms/step - loss: 0.6230 - mean_absolute_error: 0.0269 - val_loss: 0.6327 - val_mean_absolute_error: 0.0406
Epoch 21/30
59/59 [==============================] - 3s 46ms/step - loss: 0.6229 - mean_absolute_error: 0.0263 - val_loss: 0.6296 - val_mean_absolute_error: 0.0247
Epoch 22/30

59/59 [==============================] - 3s 46ms/step - loss: 0.6227 - mean_absolute_error: 0.0255 - val_loss: 0.6305 - val_mean_absolute_error: 0.0306
Epoch 23/30

59/59 [==============================] - ETA: 0s - loss: 0.6228 - mean_absolute_error: 0.0261
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 72ms/step - loss: 0.6228 - mean_absolute_error: 0.0261 - val_loss: 0.6293 - val_mean_absolute_error: 0.0228
Epoch 24/30
33/59 [===============>..............] - ETA: 1s - loss: 0.6227 - mean_absolute_error: 0.0240
59/59 [==============================] - 3s 47ms/step - loss: 0.6225 - mean_absolute_error: 0.0240 - val_loss: 0.6293 - val_mean_absolute_error: 0.0231
Epoch 25/30

59/59 [==============================] - ETA: 0s - loss: 0.6225 - mean_absolute_error: 0.0241
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
59/59 [==============================] - 4s 69ms/step - loss: 0.6225 - mean_absolute_error: 0.0241 - val_loss: 0.6293 - val_mean_absolute_error: 0.0225
Epoch 26/30
15/59 [======>.......................] - ETA: 1s - loss: 0.6207 - mean_absolute_error: 0.0232

59/59 [==============================] - ETA: 0s - loss: 0.6224 - mean_absolute_error: 0.0241
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
59/59 [==============================] - 9s 151ms/step - loss: 0.6224 - mean_absolute_error: 0.0241 - val_loss: 0.6292 - val_mean_absolute_error: 0.0223
Epoch 27/30
59/59 [==============================] - 3s 45ms/step - loss: 0.6225 - mean_absolute_error: 0.0246 - val_loss: 0.6297 - val_mean_absolute_error: 0.0257
Epoch 28/30
59/59 [==============================] - 3s 46ms/step - loss: 0.6226 - mean_absolute_error: 0.0252 - val_loss: 0.6293 - val_mean_absolute_error: 0.0234
Epoch 29/30

59/59 [==============================] - 3s 48ms/step - loss: 0.6227 - mean_absolute_error: 0.0259 - val_loss: 0.6299 - val_mean_absolute_error: 0.0275
Epoch 30/30

59/59 [==============================] - 3s 44ms/step - loss: 0.6226 - mean_absolute_error: 0.0248 - val_loss: 0.6296 - val_mean_absolute_error: 0.0252